libs/jsonschema/extension.go

```diff
@@ -26,4 +26,8 @@ type Extension struct {
 	// If the CLI version is less than this value, then validation for this
 	// schema will fail.
 	MinDatabricksCliVersion string `json:"min_databricks_cli_version,omitempty"`
+
+	// Version of the schema. This is used to determine if the schema is
+	// compatible with the current CLI version.
+	Version *int `json:"version,omitempty"`
```

Comment:
Why can't we use `MinDatabricksCliVersion` for this? They seem to be serving the same purpose

Reply:
But can it be done with let's say having `MaxDatabricksCliVersion ` later to allow for this?

Reply:
Yeah, but that's not a great UX. Template authors should not have to track CLI versions supported by their template.  A better contract is for them to subscribe to a template version.

I agree though in retrospect that maybe we could have done this in the first place and not added `MinDatabricksCliVersion`